plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/dataexport/ResourceTypeExporter.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2025 ANURAG SISODIYA
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   ANURAG SISODIYA - Initial implementation of ResourceTypeExporter
+ *******************************************************************************/
+
+package org.eclipse.fordiac.ide.model.dataexport;
+
+import javax.xml.stream.XMLStreamException;
+
+import org.eclipse.fordiac.ide.model.LibraryElementTags;
+import org.eclipse.fordiac.ide.model.libraryElement.FBNetwork;
+import org.eclipse.fordiac.ide.model.libraryElement.ResourceType;
+import org.eclipse.fordiac.ide.model.libraryElement.VarDeclaration;
+
+public class ResourceTypeExporter extends AbstractTypeExporter {
+
+	public ResourceTypeExporter(final ResourceType type) {
+		super(type);
+	}
+
+	@Override
+	protected String getRootTag() {
+		return LibraryElementTags.RESOURCETYPE_ELEMENT;
+	}
+
+	@Override
+	protected void createXMLEntries() throws XMLStreamException {
+		getWriter().writeDTD(LINE_END
```

Comment:
I just noticed that Github lost my comment to this line. For all the other elements we are normally not exporting the `DOCTYPE` entry anymore. Therefore I would find it better to not have it for resources as well.